File: algo-2/latihan/bab-3/latihan3_1_5.py
print("Program Menentukan Bilangan yang Terkecil")
print("*****************************************")

a = int(input("Masukkan bilangan ke-1: "))
b = int(input("Masukkan bilangan ke-2: "))
c = int(input("Masukkan bilangan ke-3: "))

print()
if a < b and a < c:
    print("Bilangan ke-1 paling kecil")
elif b < a and b < c:
    print("Bilangan ke-2 paling kecil")
elif c < a and c < b:
    print("Bilangan ke-3 paling kecil")
else:
    print("Ada dua atau tiga masukkan memiliki nilai sama")

File: algo-2/latihan-haskell/bab-3/latihan3_1_5.hs
main :: IO ()
main = do
  putStrLn "Program Menentukan Bilangan yang Terkecil"
  putStrLn "*****************************************"

  putStr "Masukkan bilangan ke-1: "
  a <- readLn :: IO Int
  putStr "Masukkan bilangan ke-2: "
  b <- readLn :: IO Int
  putStr "Masukkan bilangan ke-3: "
  c <- readLn :: IO Int

  putStr "\n"

  putStrLn (check a b c)
  where
    check a b c
      | a < b && a < c = "Bilangan ke-1 paling kecil"
      | b < a && b < c = "Bilangan ke-2 paling kecil"
      | c < a && c < b = "Bilangan ke-3 paling kecil"
      | otherwise = "Ada dua atau tiga masukkan memiliki nilai sama"
